masterlist.yaml

```diff
@@ -8750,7 +8750,6 @@ plugins:
         util: 'TES4Edit v4.0.4'
 
 ###### Gameplay - Character Classes & Races ######
-  ### Race Balancing Project ###
```

Comment:
You can keep the other subsections headers since they help organize the large number of plugins from this mod, but this header can go.